LibraBFT/Impl/Consensus/Liveness/RoundState.agda

```diff
@@ -41,13 +43,28 @@ processLocalTimeoutM now obmEpoch round = do
 
 ------------------------------------------------------------------------------
 
+maybeAdvanceRound : Round → SyncInfo → Maybe (Round × NewRoundReason)
+
 processCertificatesM : Instant → SyncInfo → LBFT (Maybe NewRoundEvent)
-processCertificatesM now syncInfo = do
+processCertificatesM now syncInfo =
+  logEE ("RoundState" ∷ "processCertificatesM" {-∷ lsSI syncInfo-} ∷ []) $ do
   rshcr <- use (lRoundState ∙ rsHighestCommittedRound)
```

Comment:
This differs from the Haskell code, which directly uses the `rsHighestCommittedRound` lens derived from the `RWRoundState` constraint.  I'm just noting here that I would like to discuss this at higher bandwidth.  I think at a minimum we should have some explanation about this.  I think it makes sense to put the explanation with the definitions of the `l*` lenses (like `lRoundState` in this case).

Reply:
I put a comment in LibraBFT.ImplShared.Consensus.Types